### DISCUSSION TITLE
`Something went wrong` from notifications

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#50721] [Mobile] [answered]
GitHub Mobile beta

1. Open GitHub Mobile
2. Switch to notifications 
     ![Screenshot_20230322-112921.png](https://user-images.githubusercontent.com/2119212/226955624-61cffe5a-9e89-4f2b-935c-e574c88223ee.png)
3. Tap the second one
     ![Screenshot_20230322-112940.png](https://user-images.githubusercontent.com/2119212/226955637-af3a7ddb-9ec8-432a-adc5-739774af9052.png)
4. Tap back
    ![Screenshot_20230322-112951.png](https://user-images.githubusercontent.com/2119212/226955677-5776d5c0-5c4a-4663-a805-7431e5127847.png)
     https://github.com/community/community/discussions/50711

Obviously `Something went wrong` is unhelpful

**Comment by aconsuegra** [comment, marked as answer]
Oh, I can totally reproduce this issue. Thanks for the report, we'll try to ship a fix as soon as possible.

@jsoref What is the version of the app that you are using? Could you also share the Android OS version and device model? Thanks!

**Reply by jsoref** [reply]
App 1.106.0
Android 12
Google Pixel 3A

**Reply by aconsuegra** [reply]
Thanks! We just shipped 1.106.1 with a potential fix for this bug. Could you double check if the bug is fixed?

**Reply by jsoref** [reply]
Seems ok. (Was it only the second notification? I didn't try to figure out the exact trigger for this bug.)
